ffi/neo4j/driver/internal_driver.rb

```diff
@@ -16,9 +19,8 @@ def initialize(uri, auth_token)
         Bolt::Config.destroy(config)
       end
 
-      def session
+      def session(&block)
```

Comment:
unnecessary change. That's what autoclosable does

Reply:
This is accidental.